README.md

```diff
@@ -89,6 +93,7 @@ Usage in an HTML (or alike) file:
 ```html
 <script src='https://cdn.jsdelivr.net/npm/meilisearch@latest/dist/bundles/meilisearch.umd.js'></script>
 <script>
+  // also works with window.MeiliSearch, meilisearch.MeiliSearch, window.meilisearch.MeiliSearch
```

Comment:
Can you remove the comment and only use the recommended way? 

Prefer using the `Meilisearch` (lowercase S) import